### ISSUE TITLE
Double confirmation window hangs IDE

### ISSUE THREAD

**Issue by BastiaanOlij** [godotengine/godot#84171]
### Godot version

4.2.dev

### System information

Windows

### Issue description

I've loaded an older project into Godot and am attempting to export this to Quest. I now get this new dialog due to having older mesh data in this project:

![image](https://github.com/godotengine/godot/assets/1945449/46be20f3-d0d0-45a9-83e6-db79e9e0ea09)

The problem is, at this point in time the dialog for exporting to Quest (/android) is already open, I get a warning that two dialogs are trying to take focus, and the UI stalls. Only force quit exists

### Steps to reproduce

Load an older project that has older mesh data, attempt to export to Android.

### Minimal reproduction project

https://github.com/godotvr/godot-xr-template

**Comment by BastiaanOlij** [comment]
cc @clayjohn

**Comment by BastiaanOlij** [comment]
I'm also getting stuck when just exporting the project from the export dialog.

This may be a way to test this for people without a headset.

![image](https://github.com/godotengine/godot/assets/1945449/7bab934a-3c63-419d-9b02-7e2a84997e20)

**Comment by BastiaanOlij** [comment]
And this is the error that briefly appears:

![image](https://github.com/godotengine/godot/assets/1945449/6efdf02e-c619-4b35-be21-b2092fc0b106)

**Comment by BastiaanOlij** [comment]
OK, this is even more weird. I found out that the file in the project that has the old data in it is this one: 
![image](https://github.com/godotengine/godot/assets/1945449/bbedf910-f5ed-4a1c-ad56-69fa23f2b896)

Opening up that scene also gives me the dialog, but again it hangs, it doesn't react to pressing the button. 

The only way I was able to proceed was to edit the code and skip the dialog and force the conversion.
